hydra_python_core/doc_writer.py

```diff
@@ -418,48 +504,48 @@ def add_Collection(self, collection: HydraCollection) -> None:
         entrypoint_collection = EntryPointCollection(collection)
         self.collections.append(entrypoint_collection.generate())
         self.entrypoint.add_supported_prop(entrypoint_collection)
-        self.context.add(entrypoint_collection.name, {
-            "@id": entrypoint_collection.id_, "@type": "@id"})
+        self.context.add(
+            entrypoint_collection.name,
+            {"@id": entrypoint_collection.id_, "@type": "@id"},
+        )
 
     def generate(self) -> Dict[str, Any]:
         """Get as a Python dict."""
         return self.entrypoint.generate()
 
     def get(self) -> Dict[str, str]:
-        """Create the EntryPoint object to be returnd for the get function."""
+        """Create the EntryPoint object to be returned for the get function."""
         object_ = {
-            "@context": "{}{}/contexts/EntryPoint.jsonld".format(self.url,self.api),
-            "@id": "{}{}".format(self.url,self.api),
+            "@context": "{}{}/contexts/EntryPoint.jsonld".format(self.url, self.api),
+            "@id": "{}{}".format(self.url, self.api),
             "@type": "EntryPoint",
-
         }
         for item in self.entrypoint.supportedProperty:
             uri = item.id_
             if item.generate() in self.collections:
                 collection_returned = item.generate()
-                collection_id = uri.replace(
-                    "{}EntryPoint".format(DocUrl.doc_url), "{}{}".format(self.url,self.api))
+                collection_id = uri
```

Comment:
Some changes that were made in HydraEntrypoint class in previous commits are resulting in breakage.